chart/values.yaml

```diff
@@ -97,18 +97,11 @@ api:
     frontendUrl: http://localhost:4000
     apiUrl: http://localhost:8080
     logLevel: info
-    jupyterRepository: docker.io
-    jupyterImage: briefer/enterprise/jupyter
-    jupyterTag: v566
-    jupyterPort: 8888
```

Comment:
We don't need this because the jupyter env is not created by the API on the OSS version, right? If so LGTM, just confirming